test/e2e/containers/config.go

```diff
@@ -17,8 +17,8 @@ const (
 	// name of babylon image produced by running `make build-docker`
 	BabylonContainerName = "babylonlabs-io/babylond"
 	// name of babylon image before the upgrade
-	BabylonContainerNameBeforeUpgrade = "babylonlabs/babylond"
-	BabylonContainerTagBeforeUpgrade  = "v1.0.1"
+	BabylonContainerNameBeforeUpgrade = "babylonlabs-io/babylond"
```

Comment:
I think the name in docker hub is still `babylonlabs/babylond`
https://hub.docker.com/r/babylonlabs/babylond